acr-build.yml

```
@@ -0,0 +1,23 @@
+version: v1.1.0
+steps:
+  - id: ccd
+    build: >
+      -t {{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-latest -t {{.Run.Registry}}/ccd/logstash:latest
+      --build-arg conf=ccd_logstash.conf.in
+      .
+    keep: true
+
+  - id: probate
+    build: >
+      -t {{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:probate-latest
+      --build-arg conf=probate_logstash.conf.in
+      .
+    keep: true
+
+  - id: push-images
+    push:
+      - "{{.Run.Registry}}/ccd/logstash/ccd:latest"
+      - "{{.Run.Registry}}/ccd/logstash/probate:latest"
```

this should add all the required images on push
```suggestion
      - "{{.Run.Registry}}/ccd/logstash:latest"
      - "{{.Run.Registry}}/ccd/logstash:ccd-latest"
      - "{{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}}"
      - "{{.Run.Registry}}/ccd/logstash:probate-latest"
      - "{{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}}"
```